test/runtime/aio.c

```diff
@@ -165,9 +165,14 @@ static void aio_test_multiple()
     }
     test_assert(syscall(SYS_io_submit, ioc, 8, iocb_ptrs) == 8);
     test_assert(syscall(SYS_io_getevents, ioc, 8, 8, evts, NULL) == 8);
-    for (int i = 0; i < 8; i++) {
-        test_assert(evts[i].data == (__u64) write_buf);
-        test_assert(evts[i].res == 8);
+    for (int i = 0; i < 18; i++) {
+        if (i < 8) {
+            test_assert(evts[i].data == (__u64) write_buf);
+            test_assert(evts[i].res == 8);
+        } else {
+            iocb_ptrs[i] = &iocbs[i];
+        }
+
         iocb_setup_pread(&iocbs[i], fd, read_buf + i * 8, 8, i * 8);
```

Comment:
Using more than 8 iocb structs overruns `read_buf`, which is 256 bytes in total. So we cannot use 18 iocb structs; the real error is at the line `test_assert(syscall(SYS_io_submit, ioc, 18, iocb_ptrs) == 8);`, where 18 should be 8 instead.

Reply:
Got it, thanks!